### ISSUE TITLE
Metadata Discussion

### ISSUE THREAD

**Issue by ghost** [linz/anzlic-validator#4]
**Issue to discuss** 

1. Criteria to determine if metadata is in error
2. Decide on metadata selection set
2.1 Additional checks eg extents, addresses

3. How to perform metadata corrective action

4. Support Dublin Core validation?

5. UI. QGis + Web
6. Up/Down load wrapper
7. Controller

8. Build/Deploy

**Comment by Ashleigh234** [comment]
This https://confluence.linz.govt.nz/pages/viewpage.action?spaceKey=GEOD&title=Publish+-+Create+metadata, shows the fields from the ANZLIC MetLite tool wanted and has a link to the XML template. 

From this some of the errors/ differences between metadata I have found are:
- Four layers don't have scope level/ hierarchy levels set as 'dataset'
- Some contain invalid date formats
- There are a few variations in contact details, differing from the XML template. 
    - Some have text in Individual Name, or contain 'omit', 'Omit' or None, when the XML template has 'Omit'
    - There is 'LINZ', 'Land Information New Zealand', or 'LINZ - Land Information New Zealand'
    - There are also different address fields for LINZ
        - Addresses - 'Land Information New Zeland, 155 The Terrace', '160 Lambton Quay, 155 the Terrace', '155 The Terrace' etc. 
        - Postcodes - '6011', and '6145'
        - Emails - 'info@linz.govt.nz', 'customersupport@linz.govt.nz'
    - There is a few spelling errors in Position Name, for example 'Data Manager Addresing' rather than 'Data Manager Addressing'
    - There are also different variations of phone numbers between the 0800 number and other phone number, and the way in which they are displayed. 
    - The above document also says the metadata role should be 'resourceProvider' and another role with 'pointOfContact'. Some contain roles of 'author' or 'principleInvestigator'. Or contain double up's of 'resourceProvider' roles. 
- The layers which have reference systems, are written in different formats. eg., '2193', urn:ogc:def:crs:EPSG::2193', 'NZGD2000 / New Zealand Transverse Mercator 2000'
- The document above says that resource use constraints should be copyright and license, but there are layers with only copyright or only license.
- There are layers that have metadata copyright and license use constraints but there is no copyright or license text. 
- Layers with invalid topic categories
- There are layers that don't contain bounding box extents which are supposed to be added when uploaded to LDS
- There are layers that don't contain extent descriptions, or saying the description extent is 'aus' rather than 'nzl'
- Some layers don't contain lineage information (not sure if it is an error, but there are only 42 layers that don't contain. 
- There are also 482 layers that all contain the same file identifier as the template.

**Comment by ghost** [comment]
Good work. I knew about a few of those but some of your findings could be significant. Copyright and license are audited so they need to be correct.

Do you think its worth it to create a report outlining this stuff? It might trigger a review and maybe turn this into a proper project

**Comment by Ashleigh234** [comment]
Yip, I could put this into a report, detailing the errors/ variations between the metadata, that way it is all in a document.

**Comment by ghost** [comment]
Exactly. Lets do that then. The planning priorities for the year are still being finalised and I think this is something that has been a problem for a while.

**Comment by Ashleigh234** [comment]
I've detailed all the errors or variations in the metadata here:
[LDS-Metadata.docx](https://github.com/linz/anzlic-validator/files/1913458/LDS-Metadata.docx)

**Comment by ghost** [comment]
Thanks. There are some interesting findings in there. I wonder for example if we are hosting certain layers on behalf of DOC or maybe we just inherited them and didn't get around to changing the metadata. The SRS is supposed to be standard with 4167, 2193 (+variations) and Antarctica gets its own. It shouldn't just be some random number. We'll need guidance on some of those from their respective data managers.

Will circulate and see what corrective action needs to be taken.

_Nobody seemed too interested in fixing these problems. Does that increase the need for an easy-to-use tool?_
